pkg/utils/utils.go

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package internal
+package utils
```

Comment:
I'd rather not introduce another `utils` package to a world already full of them. These tend to grow into weird collections of everything and grow unnecessary dependencies. It's not so bad when it's `internal` and the only consumers we have to care about untangling are ourselves, but making it public makes it harder to undo our mistakes in the future.

Can we either:
a. find a better name for this package, scoped to cert utilities? (not `certutils`, naming is hard)
b. find a place for this in sigstore/sigstore, since this isn't really unique to gitsign anyway, AFAIK
c. just recommend that folks copypaste this relatively small amount of code, instead of taking a dependency? The risk there is that if this logic ever changes in breaking ways, old forks in fetchit, etc., will need to update accordingly.

Reply:
I'll leave it as internal for now w/ cp/paste, and moving to sigstore/sigstore makes sense

Reply:
updated, removed utils pkg and moved this back to internal/utils.go

Reply:
Thanks!